cmake/config.cmake

```diff
@@ -42,6 +42,13 @@ set(USE_ROCM OFF)
 # Whether enable SDAccel runtime
 set(USE_SDACCEL OFF)
 
+# Whether enable Intel FPGA SDK for OpenCL (AOCL) during compile,
+#
+# Possible values:
+# - OFF: disbale AOCL
+# - board_name: use specific board name for offline compilation
```

Comment:
I think the board name should be passed with target options (e.g. `tvm.context("opencl -device=[board name]")`).

Reply:
Changed to use target name option like tgt="aocl -device=de5net_a7".